src/translation/translation_ja_JP.ts

```diff
@@ -0,0 +1,4341 @@
+    </message>
+    <message>
+        <location filename="../util.cpp" line="421"/>
+        <source>Flag icons by Mark James</source>
+        <translation>国旗アイコン by Mark James</translation>
+    </message>
+    <message>
+        <source>Some sound samples are from</source>
+        <translation type="vanished">一部のサウンドサンプルは</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="417"/>
+        <source>Some pixmaps are from the %1</source>
+        <translation>一部の画像は %1 から取得しています</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="425"/>
+        <source>Some sound samples are from %1</source>
+        <translation>一部のサウンドサンプルは %1 から取得しています</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="504"/>
+        <source>For details on the contributions check out the %1</source>
+        <translation>貢献の詳細については %1 を確認してください</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="505"/>
+        <source>Github Contributors list</source>
+        <translation>Github 貢献者リスト</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="508"/>
+        <source>Spanish</source>
+        <translation>スペイン語</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="512"/>
+        <source>French</source>
+        <translation>フランス語</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="517"/>
+        <source>Portuguese</source>
+        <translation>ポルトガル語</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="524"/>
+        <source>Dutch</source>
+        <translation>オランダ語</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="529"/>
+        <source>Italian</source>
+        <translation>イタリア語</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="533"/>
+        <source>German</source>
+        <translation>ドイツ語</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="538"/>
+        <source>Polish</source>
+        <translation>ポーランド語</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="543"/>
+        <source>Swedish</source>
+        <translation>スウェーデン語</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="549"/>
+        <source>Korean</source>
+        <translation>韓国語</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="554"/>
+        <source>Slovak</source>
+        <translation>スロバキア語</translation>
+    </message>
+    <message>
+        <location filename="../util.cpp" line="557"/>
+        <source>Simplified Chinese</source>
+        <translation>簡体字中国語</translation>
```

Comment:
Is this valid? I don't understand Chinese or Japanese, but these signs look chinese to me.